### DISCUSSION TITLE
NoahMP 4.0.1 crop modeling related parameters?

### DISCUSSION THREAD

**Discussion by yxue4** [NASA-LIS/LISF#827] [Q&A] [answered]
Dear LIS team,

I was trying to explore the usage of crop modeling within NoahMP 4.0.1. I saw that NoahMP 4.0.1 (when enabled with crop modeling) requires reading “HARVEST”, “PLANTING”, and “SEASON_GDD” information (acquired from LDT) within LIS. 

But I do not see the current version of LDT writes in “HARVEST”, “PLANTING”, AND “SEASON_GDD” information. Did I miss anything here? Please feel free to correct me.

In addition, would you please advise where to obtain “HARVEST”, “PLANTING”, and “SEASON_GDD” related information? I was not able to find such information in the manual.

Thank you so much for your help!

Best,
Yuan

**Comment by dmocko** [comment, marked as answer]
Hi Yuan,

Thanks for reaching out.  No, you didn't miss anything, as LDT currently does not process these crop variables required for Noah-MP-4.0.1 LSM.

If you look at the LIS Users' Guide, you can see that the Noah-MP-4.0.1 crop model physics options are currently not supported.

The code to run the crop model in LIS-Noah-MP-4.0.1 is generally included in the release, but has not been tested at all.

However, as you note, the ability to read/process these crop parameters will need to be added to LDT.

Please reach out to the Noah-MP developers on where to obtain these crop parameters.  Some of these may only be available within CONUS.

David
